subjects/java/checkpoints/unit-converter/README.md

```diff
@@ -0,0 +1,55 @@
+## UnitConverter
+
+### Instructions
+
+In a file named `UnitConverter.java`, write a function `convert` that returns the result of a unit conversion specified in the parameters. The function should handle different types of unit conversions based on the input parameters which specify the unit from, the unit to, and the value to be converted.
+
+##### Request conversions
+
+- Fahrenheit to Celsius: (F - 32) \* 5/9
+- Celsius to Fahrenheit: C \* 9/5 + 32
+- Kilometers to Miles: K \* 0.621371
+- Miles to Kilometers: M \* 1.60934
+
+### Expected Functions
+
+```java
+public class UnitConverter {
+    public static String convert(String[] args) {
+        // your code here
+    }
+}
+```
+
+### Usage
+
+Here is a possible `ExerciseRunner.java` to test your function :
+
+```java
+public class ExerciseRunner {
+    public static void main(String[] args) {
+        System.out.println(UnitConverter.convert(new String[]{"celsius", "fahrenheit", "100"}));
+        System.out.println(UnitConverter.convert(new String[]{"fahrenheit", "celsius", "212"}));
+        System.out.println(UnitConverter.convert(new String[]{"kilometers", "miles", "5"}));
+        System.out.println(UnitConverter.convert(new String[]{"pounds", "kilograms", "10"}));
+        if (args.length == 3) {
```

Comment:
What are the lines below supposed to showcase?
When showcasing the execution no arguments are passed to the program + it is requested to create a function and not a program

Reply:
was added it just to test and i forget to remove them